packages/redux/src/exchanges/reducer.ts

```diff
@@ -0,0 +1,128 @@
+import * as actionTypes from './actionTypes.js';
+import { type AnyAction, combineReducers, type Reducer } from 'redux';
+import { createReducerWithResult } from '../helpers/reducerFactory.js';
+import { LOGOUT_SUCCESS } from '../users/authentication/actionTypes.js';
+import type * as T from './types/index.js';
+
+export const INITIAL_STATE: T.ExchangesState = {
+  error: null,
+  isLoading: false,
+  result: null,
+  exchangeFilters: {
+    result: null,
+    error: null,
+    isLoading: false,
+  },
+  exchangeBookRequest: {
+    result: null,
+    error: null,
+    isLoading: false,
+  },
+};
+
+const error = (state = INITIAL_STATE.error, action: AnyAction) => {
```

Comment:
error slice should only take care of a single entity, so in this case it should manage the `Exchange` entity requests and Exchange Book Request and Exchange Filters requests should be managed by its corresponding slices.

The same for the `isLoading` slice.

Reply:
Done. Can you check if it's correct now?